x/circuit/keeper/msg_server_test.go

```diff
@@ -194,7 +194,7 @@ func TestTripCircuitBreaker(t *testing.T) {
 	require.ErrorContains(t, err, "MsgEditValidator: unauthorized")
 
 	// user tries to trip an already tripped circuit breaker
-	alreadyTripped := "cosmos.bank.v1beta1.MsgSend"
+	alreadyTripped := msgSend
```

Comment:
Maybe im missing something but where is msgSend set then

Reply:
It's set somewhere above, got that in the linter saying something like "you are using that string too many times while you've already declared a constant"